tmt/steps/provision/mrack.py

```diff
@@ -1039,6 +1077,130 @@ class BeakerGuestData(tmt.steps.provision.GuestSshData):
              """,
     )
 
+    bootc_user: Optional[str] = field(
+        default=None,
+        option=('--bootc-user'),
+        metavar='BOOTC_USER',
```

Comment:
Metavars usually describe the type of data, and it would be the same as with `--user`, i.e. `NAME` should be enough.